DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -52,10 +55,12 @@ class TabSwitcherViewController: UIViewController {
         currentSelection = tabsModel.currentIndex
         applyTheme(ThemeManager.shared.currentTheme)
         becomeFirstResponder()
+        tipsEnabled = tipsStorage.isEnabled
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        tipsStorage.isEnabled = false
```

Comment:
Only concern about this, is that it is persisted and in case user kills the app (or leaves it, so it is then killed by the system) while being on Tabs screen, then no pending tips will be shown.

Reply:
The ideal situation would be if the tab switcher caused "viewDidDisappear" to happen on the parent controller but it doesn't because we need the underlying context due to it being semi transparent.  Any other ideas (short of redesigning the entire tab switcher :) )?

Reply:
Feels convoluted, but I'm going to use the TabSwitcher delegate to call back up to the MainVC then down in to the TabVC to install/remove tips.